src/components/Answer.vue

```diff
@@ -319,9 +319,11 @@ import {
   commitSetWorkingDraft,
 } from '@/store/main/mutations';
 import { apiComment } from '@/api/comment';
+import BaseCard from '@/components/base/BaseCard.vue';
 
 @Component({
   components: {
+    BaseCard,
```

Comment:
Where is it used?

Reply:
should delete it, miss out it when i roll back the code